### PR TITLE
wxmac 3.1.3

### PR DESCRIPTION
**Pull request by mcandre**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Need help getting newer wxwidgets v3.1.0 to compile, so that BOINC developers can start using this formula. The current build error is:

```console
$ brew install -s wxwidgets
fatal: Needed a single revision
invalid upstream 'origin/master'
==> Downloading https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/1764
######################################################################## 100.0%
==> Patching
patching file src/osx/carbon/dcclient.cpp
Hunk #1 succeeded at 149 (offset -40 lines).
==> ./configure --prefix=/usr/local/Cellar/wxmac/3.1.0_2 --enable-clipboard --enable-cont
==> make install
Last 15 lines from /Users/andrew/Library/Logs/Homebrew/wxmac/02.make:
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_utils_osx.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/utils_osx.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_window_osx.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/window_osx.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_core_bitmap.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/bitmap.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_core_colour.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/colour.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_core_dcmemory.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/dcmemory.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_core_display.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/display.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_core_fontenum.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/fontenum.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_hid.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/hid.cpp
/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/bk-deps clang++ -mmacosx-version-min=10.15 -c -o coredll_printmac.o  -D__WXOSX_COCOA__      -DWXBUILDING      -I./src/regex  -DWXUSINGDLL -DWXMAKINGDLL_CORE -DwxUSE_BASE=0 -dynamic -fPIC -DPIC -Wall -Wundef -Wunused-parameter -Wno-ctor-dtor-privacy -Woverloaded-virtual -Wno-deprecated-declarations -D_FILE_OFFSET_BITS=64 -I/private/tmp/wxmac-20200326-41381-1h92oys/wxWidgets-3.1.0/lib/wx/include/osx_cocoa-unicode-3.1 -I./include -O2 -fno-common -fvisibility=hidden -fvisibility-inlines-hidden ./src/osx/core/printmac.cpp
./src/osx/core/bitmap.cpp:977:5: error: use of undeclared identifier 'verify_noerr'
    verify_noerr( AcquireIconRef(icon) );
    ^
1 error generated.
make: *** [coredll_core_bitmap.o] Error 1
make: *** Waiting for unfinished jobs....

READ THIS: https://docs.brew.sh/Troubleshooting
```

### CODE REVIEW

**Comment by dawidd6** [comment]
Try removing the patch.

File: Formula/wxmac.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Wxmac < Formula
   desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS)"
   homepage "https://www.wxwidgets.org"
-  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2"
-  sha256 "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2"
+  sha256 "e082460fb6bf14b7dd6e8ac142598d1d3d0b08a7b5ba402fdbf8711da7e66da8"
   revision 2
```
Comment (bayandin) [inline Formula/wxmac.rb]:
```suggestion
  revision 2
```

File: Formula/wxmac.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Wxmac < Formula
   desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS)"
   homepage "https://www.wxwidgets.org"
-  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2"
-  sha256 "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2"
+  sha256 "e082460fb6bf14b7dd6e8ac142598d1d3d0b08a7b5ba402fdbf8711da7e66da8"
```
Comment (bayandin) [inline Formula/wxmac.rb]:
```suggestion
  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.3/wxWidgets-3.1.3.tar.bz2"
  sha256 "fffc1d34dac54ff7008df327907984b156c50cff5a2f36ee3da6052744ab554a"
```

**Review by bayandin — CHANGES_REQUESTED** [review]

File: Formula/wxmac.rb (hunk 1 above)
Comment (mcandre) [reply Formula/wxmac.rb]: Can you help me to understand this change. Is this about indentation or possibly line terminators?

Comment (bayandin) [reply Formula/wxmac.rb]: It looks the diff was screwed somehow
You need to delete `revision 2`.

File: Formula/wxmac.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class Wxmac < Formula
   desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS)"
   homepage "https://www.wxwidgets.org"
-  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2"
-  sha256 "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.3/wxWidgets-3.1.3.tar.bz2"
+  sha256 "fffc1d34dac54ff7008df327907984b156c50cff5a2f36ee3da6052744ab554a"
   revision 2
```
Comment (bayandin) [inline Formula/wxmac.rb]:
```suggestion
```

Here is the proper one

**Comment by bayandin** [comment]
If you're deleting a patch itself, you also need to delete it applying of these patch:

```ruby
  # Adjust assertion which fails for wxGLCanvas due to changes in macOS 10.14.
  # Patch taken from upstream WX_3_0_BRANCH:
  # https://github.com/wxWidgets/wxWidgets/commit/531fdbcb64b265e6f24f1f0cc7469f308b9fb697
  patch :DATA
```

**Comment by bayandin** [comment]
Also, may I ask you not to rush commits and try to build it locally first.

File: Formula/wxmac.rb (hunk 1 above)
Comment (mcandre) [reply Formula/wxmac.rb]: Done.

File: Formula/wxmac.rb (hunk 3 above)
Comment (mcandre) [reply Formula/wxmac.rb]: Done.

**Comment by mcandre** [comment]
> Also, may I ask you @mcandre not to rush commits and try to build it locally first.

What is your workflow for this? I'm mad dashing force pushing and updating the submodule in my /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core local directory to test.

**Comment by mcandre** [comment]
Yeah, we fixed the compilation error. Thanks all for your support!

**Comment by issyl0** [comment]
> What is your workflow for this? I'm mad dashing force pushing and updating the submodule in my /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core local directory to test.

If I understand correctly, you've got a separate checkout of this repo elsewhere on your system that you're pushing from? You can skip that bit: Homebrew is just `git`, so `cd $(brew --repo homwbrew/core)` will put you into `/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core` and you can make all your changes with `brew edit` there, install with `brew install` as you would any other package, then push to your fork's branch once you've added it as a remote. I hope that helps!

**Comment by Bo98** [comment]
Because this is ABI incompatible, the following require revision bumps (one per commit - or just use `brew bump-revision --message="for wxmac" <formula>` and everything is handled for you):

* `diff-pdf`
* `erlang`
* `erlang@20`
* `erlang@21`
* `gambit`
* `homeworlds`
* `scummvm-tools`
* `spatialite-gui`
* `wxmaxima`

**Comment by bayandin** [comment]
- `erlang`
```
2020-04-17T12:14:51.9480500Z checking whether kernel poll support should be enabled... configure: error: no; kernel poll support requested but not found
2020-04-17T12:14:51.9483120Z configure: error: /private/tmp/erlang@20-20200417-90166-rfe3il/otp-OTP-20.3.8.26/erts/configure failed for erts
```

- `erlang@21`
```
2020-04-17T12:23:19.0833510Z clang -m64  -Wl,-no_weak_imports -o /private/tmp/erlang@21-20200417-4223-6d18ye/otp-OTP-21.3.8.14/bin/x86_64-apple-darwin19.4.0/erl_child_setup \
2020-04-17T12:23:19.0841980Z                   -Werror=undef -Werror=implicit -Werror=return-type  -fno-stack-check -I/private/tmp/erlang@21-20200417-4223-6d18ye/otp-OTP-21.3.8.14/erts/x86_64-apple-darwin19.4.0   -DHAVE_CONFIG_H -Wall -Wstrict-prototypes -Wmissing-prototypes -Wdeclaration-after-statement -Ibeam -Isys/unix -Isys/common -Ix86_64-apple-darwin19.4.0 -Ipcre -Ihipe -I../include -I../include/x86_64-apple-darwin19.4.0 -I../include/internal -I../include/internal/x86_64-apple-darwin19.4.0 obj/x86_64-apple-darwin19.4.0/opt/smp/erl_child_setup.o obj/x86_64-apple-darwin19.4.0/opt/smp/sys_uds.o obj/x86_64-apple-darwin19.4.0/opt/smp/hash.o -L../lib/internal/x86_64-apple-darwin19.4.0 -lerts_internal  
2020-04-17T12:23:19.0843830Z ld: weak import of symbol '___darwin_check_fd_set_overflow' not supported because of option: -no_weak_imports for architecture x86_64
2020-04-17T12:23:19.0845060Z clang: error: linker command failed with exit code 1 (use -v to see invocation)
2020-04-17T12:23:19.0846920Z make[4]: *** [/private/tmp/erlang@21-20200417-4223-6d18ye/otp-OTP-21.3.8.14/bin/x86_64-apple-darwin19.4.0/erl_child_setup] Error 1
2020-04-17T12:23:19.0847670Z make[4]: *** Waiting for unfinished jobs....
2020-04-17T12:23:19.0847860Z make[3]: *** [opt] Error 2
2020-04-17T12:23:19.0848020Z make[2]: *** [opt] Error 2
2020-04-17T12:23:19.0913890Z make[1]: *** [smp] Error 2
2020-04-17T12:23:19.0914550Z make: *** [emulator_profile_generate] Error 2
```

- `gambit`
```
2020-04-17T12:27:48.5421970Z clang++ -DHAVE_CONFIG_H -I.  -I./src -I./library/include -I./src/labenski/include -I/usr/local/lib/wx/include/osx_cocoa-unicode-3.1 -I/usr/local/include/wx-3.1 -D_FILE_OFFSET_BITS=64 -DWXUSINGDLL -D__WXMAC__ -D__WXOSX__ -D__WXOSX_COCOA__    -g -O2 -c -o src/labenski/src/sheetsel.o src/labenski/src/sheetsel.cpp
2020-04-17T12:27:48.5422390Z In file included from src/labenski/src/sheet.cpp:39:
2020-04-17T12:27:48.5422620Z ./src/labenski/include/wx/sheet/sheetspt.h:185:5: error: C++ requires a type specifier for all declarations
2020-04-17T12:27:48.5422870Z     WX_DECLARE_CONTROL_CONTAINER();
2020-04-17T12:27:48.5423120Z     ^
2020-04-17T12:27:48.5425710Z src/labenski/src/sheet.cpp:7101:9: error: use of undeclared identifier 'wxPrintf'
2020-04-17T12:27:48.5426010Z         wxPrintf(wxT("COUNT MISMATCH ERROR! \n"));
2020-04-17T12:27:48.5426210Z         ^
2020-04-17T12:27:48.5429000Z src/labenski/src/sheet.cpp:7104:31: error: use of undeclared identifier 'wxPrintf'; did you mean 'wxPoint'?
2020-04-17T12:27:48.5429330Z         if (ans[n] != res[n]) wxPrintf(wxT("Error in item %u\n"), n);
2020-04-17T12:27:48.5429560Z                               ^
2020-04-17T12:27:48.5431880Z /usr/local/include/wx-3.1/wx/gdicmn.h:563:24: note: 'wxPoint' declared here
2020-04-17T12:27:48.5432160Z class WXDLLIMPEXP_CORE wxPoint
2020-04-17T12:27:48.5432360Z                        ^
2020-04-17T12:27:48.5434440Z src/labenski/src/sheet.cpp:7106:5: error: use of undeclared identifier 'wxPrintf'
2020-04-17T12:27:48.5434760Z     wxPrintf(msg + wxT("\n"));
2020-04-17T12:27:48.5434960Z     ^
2020-04-17T12:27:48.5435190Z 4 errors generated.
2020-04-17T12:27:48.5436780Z make[1]: *** [src/labenski/src/sheet.o] Error 1
2020-04-17T12:27:48.5437280Z make[1]: *** Waiting for unfinished jobs....
2020-04-17T12:27:48.5438620Z make: *** [install-recursive] Error 1
```

- `spatialite-gui`
```
2020-04-17T12:39:16.2438140Z MalformedGeoms.cpp:1130:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2439240Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 0, 1);
2020-04-17T12:39:16.2439490Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2440490Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2440740Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2440950Z     ^
2020-04-17T12:39:16.2441150Z MalformedGeoms.cpp:1134:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2442210Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 0, 2);
2020-04-17T12:39:16.2442420Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2443530Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2443970Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2444200Z     ^
2020-04-17T12:39:16.2444400Z MalformedGeoms.cpp:1149:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2445450Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 1, 2);
2020-04-17T12:39:16.2445680Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2446810Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2447030Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2447240Z     ^
2020-04-17T12:39:16.2447450Z MalformedGeoms.cpp:1162:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2448480Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 2, 1);
2020-04-17T12:39:16.2448720Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2449820Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2450070Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2450720Z     ^
2020-04-17T12:39:16.2450910Z MalformedGeoms.cpp:1166:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2451960Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 2, 2);
2020-04-17T12:39:16.2452200Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2453310Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2453570Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2453950Z     ^
2020-04-17T12:39:16.2454160Z MalformedGeoms.cpp:1181:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2455220Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 3, 1);
2020-04-17T12:39:16.2455460Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2456580Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2456810Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2457040Z     ^
2020-04-17T12:39:16.2457240Z MalformedGeoms.cpp:1185:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2458300Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 3, 2);
2020-04-17T12:39:16.2458500Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2459610Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2459870Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2460070Z     ^
2020-04-17T12:39:16.2460280Z MalformedGeoms.cpp:1200:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2461640Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 4, 1);
2020-04-17T12:39:16.2461920Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2463030Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2463300Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2463490Z     ^
2020-04-17T12:39:16.2463860Z MalformedGeoms.cpp:1204:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2464940Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 4, 2);
2020-04-17T12:39:16.2465170Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2466190Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2466440Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2466650Z     ^
2020-04-17T12:39:16.2466850Z MalformedGeoms.cpp:1219:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2467880Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 5, 2);
2020-04-17T12:39:16.2468090Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2469160Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2469430Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2469640Z     ^
2020-04-17T12:39:16.2469830Z MalformedGeoms.cpp:1234:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2470820Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 6, 1);
2020-04-17T12:39:16.2471050Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2472140Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2472390Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2472580Z     ^
2020-04-17T12:39:16.2472800Z MalformedGeoms.cpp:1238:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2473990Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 6, 2);
2020-04-17T12:39:16.2474220Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2475280Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2475970Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2476170Z     ^
2020-04-17T12:39:16.2476370Z MalformedGeoms.cpp:1253:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2477400Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 7, 1);
2020-04-17T12:39:16.2477640Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2478680Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2478930Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2479140Z     ^
2020-04-17T12:39:16.2479330Z MalformedGeoms.cpp:1257:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2480360Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 7, 2);
2020-04-17T12:39:16.2480590Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2481710Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2481940Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2482140Z     ^
2020-04-17T12:39:16.2482350Z MalformedGeoms.cpp:1272:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2483350Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 8, 2);
2020-04-17T12:39:16.2483570Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2484830Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:16.2485090Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:16.2485300Z     ^
2020-04-17T12:39:16.2485890Z MalformedGeoms.cpp:1287:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:16.2486980Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 9, 1);
2020-04-17T12:39:16.2487220Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:16.2488300Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:25.4332620Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:25.4333070Z     ^
2020-04-17T12:39:25.4333400Z MalformedGeoms.cpp:1291:49: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:25.4335160Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 9, 2);
2020-04-17T12:39:25.4335410Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
2020-04-17T12:39:25.4336120Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:25.4336820Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:25.4337130Z     ^
2020-04-17T12:39:25.4339000Z MalformedGeoms.cpp:1305:50: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:25.4340510Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 10, 1);
2020-04-17T12:39:25.4340810Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                     ^
2020-04-17T12:39:25.4342800Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:25.4343140Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:25.4343430Z     ^
2020-04-17T12:39:25.4345420Z MalformedGeoms.cpp:1309:50: error: too few arguments to function call, expected 4, have 3
2020-04-17T12:39:25.4347190Z   GridCtrl->SetCellAlignment(wxALIGN_RIGHT, 10, 2);
2020-04-17T12:39:25.4347460Z   ~~~~~~~~~~~~~~~~~~~~~~~~~~                     ^
2020-04-17T12:39:25.4348860Z /usr/local/include/wx-3.1/wx/generic/grid.h:1592:5: note: 'SetCellAlignment' declared here
2020-04-17T12:39:25.4349890Z     void     SetCellAlignment( int row, int col, int horiz, int vert );
2020-04-17T12:39:25.4350790Z     ^
2020-04-17T12:39:25.4352340Z fatal error: too many errors emitted, stopping now [-ferror-limit=]
2020-04-17T12:39:25.4352630Z 20 errors generated.
2020-04-17T12:39:25.4353370Z make[1]: *** [MalformedGeoms.o] Error 1
2020-04-17T12:39:25.4353660Z make[1]: *** Waiting for unfinished jobs....
2020-04-17T12:39:25.4355000Z mv -f .deps/Exif.Tpo .deps/Exif.Po
2020-04-17T12:39:25.4356010Z mv -f .deps/BlobExplorer.Tpo .deps/BlobExplorer.Po
2020-04-17T12:39:25.4356920Z mv -f .deps/DialogsGraph.Tpo .deps/DialogsGraph.Po
2020-04-17T12:39:25.4358010Z mv -f .deps/Dialogs.Tpo .deps/Dialogs.Po
2020-04-17T12:39:25.4358970Z mv -f .deps/Main.Tpo .deps/Main.Po
2020-04-17T12:39:25.4359820Z make: *** [install-recursive] Error 1
```

**Comment by Bo98** [comment]
erlang@20 and erlang@21 are pre-existing failures. They do not build on Xcode 11.4.

**Pull request closed without merging**